.github/workflows/linux.yml

```diff
@@ -311,7 +311,7 @@ jobs:
           mkdir build
           cd build
           cmake -DCMAKE_INSTALL_PREFIX=$HDF5_DIR -DHDF5_ENABLE_PARALLEL=ON -DHDF5_ENABLE_THREADSAFE=ON \
-            -DALLOW_UNSUPPORTED=ON -DBUILD_TESTING=OFF -DHDF5_BUILD_HL_LIB=OFF \
+            -DHDF5_ENABLE_DIRECT_VFD=ON -DBUILD_TESTING=OFF -DHDF5_BUILD_HL_LIB=OFF \
```

Comment:
Should this be -DHDF5_ALLOW_UNSUPPORTED also?

Reply:
Missed that, thanks. Fixed.

Reply:
Missed “=ON”